hummingbot/connector/exchange/ascend_ex/ascend_ex_order_book_tracker.py

```diff
@@ -75,8 +74,8 @@ async def _track_single_book(self, trading_pair: str):
                     message = await message_queue.get()
 
                 if message.type is OrderBookMessageType.DIFF:
-                    bids, asks = active_order_tracker.convert_diff_message_to_order_book_row(message)
-                    order_book.apply_diffs(bids, asks, message.update_id)
+                    # bids, asks = active_order_tracker.convert_diff_message_to_order_book_row(message)
+                    order_book.apply_diffs(message.bids, message.asks, message.update_id)
```

Comment:
`apply_diffs()` expects bids and asks to be `List[OrderBookRow]`. I have yet to test it myself, but I am expecting an error when accessing the `update_id` attribute in the `OrderBookRow`